app/models/manageiq/providers/nuage/network_manager/event_catcher/stream.rb

```diff
@@ -30,7 +30,7 @@ def start(&message_handler_block)
     $nuage_log.debug("#{self.class.log_prefix} Opening amqp connection using options #{@options}")
     @options[:message_handler_block] = message_handler_block if message_handler_block
     with_fallback_urls(@options[:urls]) do
-      connection.run
+      run_with_idle_timeout_ignored
```

Comment:
I would not name the method like this. Isn't `run` ok and implement whatever logic there is needed? Imagine we will realise there is another error message that we ignore and you will have to rename it to `run_with_iddle_timeout_and_one_other_error_ignored` 😄.